docs/it/come-affrontare-l-eta-nel-mondo-it.md

```diff
@@ -0,0 +1,76 @@
+
+Questi ruoli sono molto interessanti e sfidanti qualora fare _solo coding_ iniziasse a non bastarti. Se vuoi gestire un team e avere responsabilità differenti, sicuramente prima o poi potrai intraprendere uno di questi percorsi senza rinunciare del tutto all'aspetto tecnico.
+
+Al di là della tua scelta, ricorda che se vuoi essere un developer che aggiunga valore nonostante l'età, non è sufficiente che ti limiti al coding, cerca sempre di contribuire a ogni aspetto del progetto: sia che tu rivesta il ruolo di developer o di team leader, impegnati costantemente nel risolvere problemi e suggerire soluzioni ottimali, senza concentrarti sui limiti assegnati dai ruoli.
+Se sei ancora junior, proponi soluzioni anche se non sei del tutto sicuro della loro correttezza. Non solo l'impegno nello studio e nell'approfondimento, ma anche e soprattutto trarre insegnamento dai propri errori è un ottimo modo per crscere.
+Così, dopo anni di esperienza, potrai mettere in campo molte conoscenze che figure più giovani del team probabilmente non avranno.
+
+Un developer che sa comunicare e sa trasmettere le proprie conoscenze ad altri, sarà sempre di grande valore per un team e avrà un'autorevolezza al suo interno a prescindere dal ruolo e soprattutto dall'età.
+
+## Il bias del fare codice a vita
+
+La carriera di chi sviluppa è complessa da molti punti di vista e non priva di incognite e, al primo posto, possiamo mettere il fatto che sia una categoria nuova e in continua evoluzione. Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante. Cercando di allontanarsi un poco per vedere un percorso di massima non possiamo non pensare alla crescita verticale verso mansioni mano a mano più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad un pensiero sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.
```

Comment:
Riuscireste a rielaborare questa frase? A mio avviso non è molto scorrevole.

"Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante"

```suggestion
La carriera di chi sviluppa è complessa sotto molti punti di vista e non priva di incognite: al primo posto possiamo assolutamente prendere coscienza del fatto che sia una categoria nuova e in continua evoluzione. Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante. Da un punto di vista più generale non possiamo ignorare la crescita verticale verso determinate mansioni più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad una considerazione sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.
```

Reply:
Non committo per non perdere il tuo commento.. Dimmi se il re-wording ti piace:
Guardando alcuni schemi che illustrano l'evoluzione e le competenze necessarie per una particolare mansione, diventa evidente di fronte a noi un albero di opzioni impressionante.

Reply:
E' proprio il concetto di albero che secondo me può creare confusione. Ho capito che fai riferimento a un albero decisionale, ma ho paura che non possa essere così scontato agli occhi di altri.

Se usassimo in qualche modo le parole "diramazioni" o "strade"?

Reply:
@AngeloAvv essendo un libro per DEV io credo che il concetto di albero sia vicino all'interlocutore

Reply:
@eppak un wannabe dev o un junior se non ha studiato algoritmi e strutture dati in modo approfondito è difficile che conosca cosa sia un albero

Reply:
@AngeloAvv però il capitolo non è improntato a uno wannabe dev.. credo possa essere sensato fare congetture di questo tipo.

Reply:
Vero, ma questo non esclude che possa volerlo comunque leggere per capire cosa aspettarsi dal futuro. Se volete lasciarlo così per me va bene la tua seconda versione @Cadienvan